<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>691</width>
      <height>511</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="WidgetPFD" name="widgetPFD" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>701</width>
        <height>461</height>
       </rect>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>HSI</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="WidgetNAV" name="widgetNAV" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>641</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>470</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>PFD</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetPFD</class>
   <extends>QWidget</extends>
   <header>WidgetPFD.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetNAV</class>
   <extends>QWidget</extends>
   <header>WidgetNAV.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
